packages/tests/run-web-tests.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @fileoverview This file replaces the WTR binary so that it can control
+ * how open ports are found, eliminating a race condition and limiting ports
+ * to a Sauce compatible list.
+ */
+
+import {startTestRunner} from '@web/test-runner';
+import config from './web-test-runner.config.js';
+
+// eslint-disable-next-line no-undef
+process.on('uncaughtException', (error) => {
+  console.log('error.code', error.code);
+  if (error.code === 'EADDRINUSE') {
+    if (currentPortIndex === SAUCE_PORTS.length) {
```

Comment:
We also check this inside startWithNextAvailablePort

Reply:
Oh yeah. This can be removed now that there is an `if` check on `EADDRINUSE` that won't match on the error thrown in the function. Nice comment!